examples/complete/main.tf

```diff
@@ -1,4 +1,27 @@
-module "example" {
-  source  = "../.."
-  example = var.example
+provider "aws" {
+  region = var.region
+}
+
+module "sns" {
+  source = "git::https://github.com/cloudposse/terraform-aws-sns-topic.git?ref=0.2.0"
```

Comment:
```suggestion
  source = "git::https://github.com/cloudposse/terraform-aws-sns-topic.git?ref=tags/0.2.0"
```

Reply:
Yeah, easy to miss. Should be converted to pre-commit/gh action check.

Reply:
https://cloudposse.atlassian.net/browse/INTERNAL-354